app/models/assignment.rb

```diff
@@ -510,38 +511,41 @@ def self.export_data(csv, scores, options)
     @scores = scores
     (0..@scores[:teams].length - 1).each do |index|
       team = @scores[:teams][index.to_s.to_sym]
+      first_participant = team[:team].participants[0] unless team[:team].participants[0].nil?
```

Comment:
write a method or search for a method that checks whether the team is empty. Team.size() can be used?